### ISSUE TITLE
Allow the compontent to be disabled with ng-disable

### ISSUE THREAD

**Issue by phillipgreenii** [alexandernst/angular-multi-select#48]
I would like to disable the component and `ng-disabled` isn't working, is there another way to disable the component?

**Comment by alexandernst** [comment]
Not really. Should be implemented. If you really need a quick (and dirty) fix, try disabling it with pure JS (or jQuery, or ...).
